app/controllers/ansible_roles_controller.rb

```diff
@@ -0,0 +1,42 @@
+#:nodoc:
```

Comment:
Please write some doc, even if they're basic, they allows us to define boundaries for our classes and make newcomers more familiar with this